fcrepo-jms/src/main/java/org/fcrepo/jms/headers/DefaultMessageFactory.java

```diff
@@ -117,6 +117,30 @@ public Message getMessage(final FedoraEvent jcrEvent,
         return message;
     }
 
+    private String getUserData(final FedoraEvent jcrEvent) {
```

Comment:
More and more, I think we should adopt a natural habit of rethrowing `RepositoryException` as soon and as low as we can. (Perhaps factored into a static utility method.) There's really nothing we expect to do with it other than eventually report it at our own API. This is to be accounted to the facts that we are composing middleware, not user-facing software, and that `RespositoryException` is generic.